tests/host_tools/uffd/src/bin/valid_handler.rs

```diff
@@ -0,0 +1,289 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I'd add a general comment about what this handler does (servicing pages one-by-one as requested)